holidays/countries/australia.py

```diff
@@ -41,6 +38,11 @@ def sovereign_birthday(self) -> str:
             else "Queen's Birthday"
         )
 
+    @property
+    def boxing_day_name(self) -> str:
```

Comment:
Do we really want a separate property for a single use case?

Reply:
Yes, it seems a bit excessive.